frontend/packages/dev-console/integration-tests/features/helm/actions-on-HelmRelease.feature

```diff
@@ -7,15 +7,35 @@ Background:
     And helm release "nodejs-ex-k" is present in topology page
 
 
+@regression, @smoke
+Scenario: Context menu options of helm release: HR-07-TC01
+    Given helm release "nodejs-ex-k" is present in topology page
```

Comment:
When this line is already available in Background section, There is no necessity to add this given statement in scenarios